src/metabase/api/card.clj

```diff
@@ -639,7 +639,7 @@
         ;; Ideally this would be DRY with the actual specification some way, but since this is a stop-gap solution, we
         ;; decided not to complicate the solution further to accomplish this.
         (search/bulk-ingest! (for [search-model ["card" "dataset" "metric"]]
-                               [search-model [:= :this.id id]])))
+                               [search-model [:= :this.id id]]) false))
```

Comment:
Having a naked boolean at the end feels a bit opaque to me, how about making it a kwarg map?

One downside would be that you're not forced to set the value, but I think it's OK having a default.

Reply:
I ended up with a combination of adding the context arg as a 1st argument, but also adjusting the search API a bit to limit the need for passing it and better isolating it to the appdb implementation of that API.

LMK what you think of the new version